apps/web/src/components/ExperienceFormFields/AwardFields.tsx

```diff
@@ -80,7 +83,15 @@ const AwardFields = ({ labels }: SubExperienceFormProps) => {
             name="issuedBy"
             type="text"
             rules={{ required: intl.formatMessage(errorMessages.required) }}
+            list="organizationSuggestions"
```

Comment:
For each of these, I'd probably conditionally add this attribute only when there are suggestions.

Reply:
Aye aye, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12427/commits/b90a9f7a8f30d7405976506ffcbfe14685358d8b
I noticed this makes the input type variable depending on whether `list` is defined, which can cause testing bugs https://github.com/GCTC-NTGC/gc-digital-talent/pull/12427/commits/36fc31fd0ce7f818922d271d0d4156d19c6eae41

Reply:
Yeah, I saw that. Makes me wonder if we should create a specific input for this. Obviously out of scope for this issue though.

Maybe once we get this merged in we can sit and stew over it for a while and see if it makes sense :woman_shrugging: